### PR TITLE
[GEN-1028] Generate failed annotations report dashboard

### PR DESCRIPTION
**Pull request by rxu17**
**Purpose:** This is a draft PR. This PR will add a section to our dashboard that showcases the number of failed annotations per center and also let sites know where to find their failed annotations report

**Testing:**
- Created R unit tests for some of the dashboard functions
- Generated the relevant section of the dashboard R markdown file. See current look:
image
- Ran the `R/dashboard_markdown_generator.R` using the relevant dashboard code above in docker image

Depends on #545

### CODE REVIEW

**Comment by rxu17** [comment]
One outstanding question is the historical reason behind not running the dashboard code on the test pipeline? Currently it defaults to running on production: https://github.com/Sage-Bionetworks/Genie/blob/446de5cd3516708b9ea728765aa5c5c6b861d680/bin/database_to_staging.py#L289-L294

**Comment by thomasyu888** [comment]
> One outstanding question is the historical reason behind not running the dashboard code on the test pipeline? Currently it defaults to running on production:
> 
> https://github.com/Sage-Bionetworks/Genie/blob/446de5cd3516708b9ea728765aa5c5c6b861d680/bin/database_to_staging.py#L289-L294

There's not a good reason other than tech debt.  If you notice in `bin/database_to_staging.py` we use `test` and `staging` parameters to choose which GENIE project to push data into, however, see this ticket; https://sagebionetworks.jira.com/browse/GEN-420.

**Review by thomasyu888 — APPROVED** [review]
🦖 LGTM!  This is great - have you had a chance to check out quarto at all?  I recently created nice looking dashboards like this using purely Python code.  It's a game changer.

Note: Not saying we should convert to quarto in this PR. ha

File: R/tests/testthat/test_dashboard_template_functions.R
Comment (thomasyu888) [inline R/tests/testthat/test_dashboard_template_functions.R:1]: Really appreciate the tests added here.

**Comment by rxu17** [comment]
> 🦖 LGTM! This is great - have you had a chance to check out quarto at all? I recently created nice looking dashboards like this using purely Python code. It's a game changer.
> 
> Note: Not saying we should convert to quarto in this PR. ha

Somehow I missed the notifications to the comments in this PR. I have not, this is my first time touching dashboard code here haha but I remember you used it for the hefty data guide PR. Quarto looks great, I am down to just convert to it (plus easier to integrate in tests when it's python and moving away from R dependencies is a plus).

**Comment by thomasyu888** [comment]
> > 🦖 LGTM! This is great - have you had a chance to check out quarto at all? I recently created nice looking dashboards like this using purely Python code. It's a game changer.
> > Note: Not saying we should convert to quarto in this PR. ha
> 
> Somehow I missed the notifications to the comments in this PR. I have not, this is my first time touching dashboard code here haha but I remember you used it for the hefty data guide PR. Quarto looks great, I am down to just convert to it (plus easier to integrate in tests when it's python and moving away from R dependencies is a plus).

@rxu17 Lets create a separate ticket to export to quarto, but this is PR is good to go

**Pull request merged**